OrbitGl/Batcher.cpp

```diff
@@ -8,7 +8,7 @@
 #include "Utils.h"
 
 void Batcher::AddLine(const Line& line, const Color* colors,
-                      PickingID::Type picking_type,
```

Comment:
comit 345da89f2a012aed0bba46d8e179046d58d57b41: please reformat commit message to title and description it is too long.

Reply:
Will do after giving you the chance for a second look so I don't need to force push before.